jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/CloudWatchLogsServiceNode.kt

```diff
@@ -42,4 +44,8 @@ class CloudWatchLogsNode(
     override fun onDoubleClick() {
         CloudWatchLogWindow.getInstance(nodeProject).showLogGroup(logGroupName)
     }
+
+    override val typeDisplayName = "Log Group"
+    override val cloudApiResourceType = "AWS::Logs::LogGroup"
```

Comment:
Should we maintain a separate list of all the resources we support that coincide with the ones in the manifest so it becomes easier to track if this resource type has been removed/added when we update the manifest?

Reply:
wouldn't that just be implementors of the `CloudApiResource` interface?